docs/website/docs/dlt-ecosystem/destinations/databricks.md

```diff
@@ -15,7 +15,85 @@ keywords: [Databricks, destination, data warehouse]
 pip install dlt[databricks]
 ```
 
-## Setup Guide
+## Set up your Databricks workspace
+
+To use the Databricks destination, you need:
+
+* A Databricks workspace with a Unity Catalog metastore connected
+* A Gen 2 Azure storage account and container
```

Comment:
It needs to be an [ADLS Gen2 storage account](https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction).